cli/start.go

```diff
@@ -394,8 +394,26 @@ func runStart(_ *cobra.Command, args []string) error {
 	for i, spec := range serverCtx.Stores.Specs {
 		fmt.Fprintf(tw, "store[%d]:\t%s\n", i, spec)
 	}
-	for i, address := range serverCtx.JoinList {
-		fmt.Fprintf(tw, "join[%d]:\t%s\n", i, address)
+	var joinStr string
+	initialBoot := s.InitialBoot()
+	nodeID := s.NodeID()
+	if initialBoot {
+		if nodeID == server.FirstNodeID {
+			fmt.Fprintf(tw, "Initialized new cluster\n")
+			joinStr = "failed to join"
```

Comment:
we can't have a non-empty `JoinList` and initialize a new cluster. This means that `failed to join` will never be printed.